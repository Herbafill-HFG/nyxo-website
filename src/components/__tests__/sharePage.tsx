import React from "react"
import renderer from "react-test-renderer"

import SharePage from "../sharePage"

describe("SharePage", () => {
  const subject = "sleep coaching is great!"
  const url = "https://www.example.com"
  const excerpt = "Lorem ipsum"

  it("renders correctly", () => {
    const tree = renderer
      .create(<SharePage subject={subject} url={url} excerpt={excerpt} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
