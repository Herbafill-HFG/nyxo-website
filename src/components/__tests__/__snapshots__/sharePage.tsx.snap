// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SharePage renders correctly 1`] = `
<div
  className="sc-gtssRu kkwQhv"
>
  <div
    className="sc-dlnjPT hujsBX"
  />
  <p
    className="sc-hKFyIo imRXgX"
  >
    Share this lesson:
  </p>
  <button
    aria-label="email"
    className="react-share__ShareButton sc-eCApGN dNxEMd"
    onClick={[Function]}
    style={
      Object {
        "backgroundColor": "transparent",
        "border": "none",
        "color": "inherit",
        "cursor": "pointer",
        "font": "inherit",
        "padding": 0,
      }
    }
  >
    <svg
      className="sc-bdnylx LxIqb"
      fill="none"
      height="30px"
      stroke="#333"
      strokeLinecap="round"
      strokeWidth="1.5"
      version="1.1"
      viewBox="0 0 30 30"
      width="30px"
      xmlns="http://www.w3.org/2000/svg"
    />
  </button>
  <button
    aria-label="linkedin"
    className="react-share__ShareButton sc-jSFkmK fdqWOx"
    onClick={[Function]}
    style={
      Object {
        "backgroundColor": "transparent",
        "border": "none",
        "color": "inherit",
        "cursor": "pointer",
        "font": "inherit",
        "padding": 0,
      }
    }
  >
    <svg
      className="sc-bdnylx LxIqb"
      fill="none"
      height="30px"
      stroke="#333"
      strokeLinecap="round"
      strokeWidth="1.5"
      version="1.1"
      viewBox="0 0 30 30"
      width="30px"
      xmlns="http://www.w3.org/2000/svg"
    />
  </button>
  <button
    aria-label="facebook"
    className="react-share__ShareButton sc-gKAblj hCOhTy"
    onClick={[Function]}
    style={
      Object {
        "backgroundColor": "transparent",
        "border": "none",
        "color": "inherit",
        "cursor": "pointer",
        "font": "inherit",
        "padding": 0,
      }
    }
  >
    <svg
      className="sc-bdnylx LxIqb"
      fill="none"
      height="30px"
      stroke="#333"
      strokeLinecap="round"
      strokeWidth="1.5"
      version="1.1"
      viewBox="0 0 30 30"
      width="30px"
      xmlns="http://www.w3.org/2000/svg"
    />
  </button>
  <button
    aria-label="twitter"
    className="react-share__ShareButton sc-iCoHVE kReqGE"
    onClick={[Function]}
    style={
      Object {
        "backgroundColor": "transparent",
        "border": "none",
        "color": "inherit",
        "cursor": "pointer",
        "font": "inherit",
        "padding": 0,
      }
    }
  >
    <svg
      className="sc-bdnylx LxIqb"
      fill="none"
      height="30px"
      stroke="#333"
      strokeLinecap="round"
      strokeWidth="1.5"
      version="1.1"
      viewBox="0 0 30 30"
      width="30px"
      xmlns="http://www.w3.org/2000/svg"
    />
  </button>
</div>
`;
